src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -38,4 +38,36 @@ public static function castDateTime(\DateTimeInterface $d, array $a, Stub $stub,
 
         return $a;
     }
+
+    public static function castInterval(\DateInterval $i, array $a, Stub $stub, $isNested, $filter)
+    {
+        $prefix = Caster::PREFIX_VIRTUAL;
+
+        $a = array(
+            $prefix.'interval' => self::formatInterval($i),
+            Caster::PREFIX_DYNAMIC.'invert' => $i->invert,
+            Caster::PREFIX_DYNAMIC.'days' => $i->days,
+        );
+
+        return $a;
+    }
+
+    private static function formatInterval(\DateInterval $i)
+    {
+        $format = '%R '
+            .($i->y ? '%yy ' : '')
+            .($i->m ? '%mm ' : '')
+            .($i->d ? '%dd ' : '')
+        ;
+
+        if (PHP_VERSION_ID >= 70100) {
```

Comment:
please use `\PHP_VERSION_ID`, to let the version comparison be applied at compile time (not qualifying the constant requires doing it at runtime to check the constant first in the current namespace)